packages/modelserver-client/src/model-server-client-api-v2.ts

```diff
@@ -70,14 +71,14 @@ export interface ModelServerClientApiV2 {
     /**
      * The `initialize` method should be executed before any other requests to the model server.
      * The given base url should point to the location of the model server API entry point.
-     * (e.g. `http://localhost:8081/api/v1/`). Once the initialization is completed the client is expected
+     * (e.g. `http://localhost:8081/api/v2/`). Once the initialization is completed the client is expected
      * to be ready and should be able to handle REST requests to & responses from the model server.
      * Any requests to the model server before the client has been initialized are expected to fail (i.e. throw an error)
      * @param baseUrl Url pointing to the API entry point
      * @param defaultFormat Optional fallback format that should used when a request method is invoked and no explicit format argument
      *                      has been passed. The default-default is `'json-v2'`
      */
-    initialize(baseUrl: string, defaultFormat?: Format): void | Promise<void>;
+    initialize(baseUrl: URI, defaultFormat?: Format): void | Promise<void>;
```

Comment:
Same comment as above about `URL` type. This obviously would cascade to call sites elsewhere in this PR.